BUILDING.md

```diff
@@ -385,11 +385,12 @@ $ backtrace
 Prerequisites:
 
 * [Python 2.6 or 2.7](https://www.python.org/downloads/)
-* The "Desktop development with C++" workload from
-  [Visual Studio 2017](https://www.visualstudio.com/downloads/) or the
-  "Visual C++ build tools" workload from the
+* The "Build Tools for Visual Studio 2017" workload from the
```

Comment:
I'll try to suggest an alternative for this whole bullet point:
```
[Visual Studio 2017](https://www.visualstudio.com/downloads/) or the	
[Build Tools for Visual Studio 2017](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2017).
Installed with the "Visual C++ build tools" Workload,
and if available the "Git for Windows" and "Python 2 32-bit (2.7.x)" Individual
Components.
```
Based on these two screens:
![image](https://user-images.githubusercontent.com/96947/48674818-691f3800-eb1e-11e8-98a3-93fe36e3ed00.png)
and
![image](https://user-images.githubusercontent.com/96947/48674848-cc10cf00-eb1e-11e8-9d77-7691323f4754.png)

Reply:
Sounds good to me 👍 
BTW do you knopw what the Node.js component adds?

Reply:
> BTW do you know what the Node.js component adds?

AFAIK it installs an LTS version (with our installer and it's defaults). Apropo that, it installs python with it's installer and it's defaults, which unfortunately do not include "Add to Path". So out-of-the-box it only works with the "Developer Command Prompt for VS 2017".

But as I show before, even if you don't select it the node binary gets in there anyway (I did not start any of those explicitly) 😜 
![image](https://user-images.githubusercontent.com/96947/48675138-3592dc80-eb23-11e8-8d6d-f093f0231845.png)

Reply:
There are two more
![image](https://user-images.githubusercontent.com/96947/48675458-fd8d9880-eb26-11e8-9fa0-bc9345739fe1.png)

ping @joshugav @Johnmont @AmandaSilver could you tell us what do the 4 above mentioned Node.js related components install, or please connect us with someone who can help?